pyjobs/core/tests/test_job_views.py

```diff
@@ -117,3 +117,24 @@ def test_check_applied_for_job(self):
         request = request_client.content.decode('utf-8')
         expected_response = "Aplicar para esta vaga pelo PyJobs"
         self.assertTrue(expected_response in request)
+
+    def test_check_if_profile_with_no_skills_can_apply(self):
+        self.client.login(username="jacob", password="top_secret")
+        job_url = "/job/{}/".format(self.job.pk)
```

Comment:
An alternative would be `job_url = f"/job/{self.job.pk}/"` or, even bettter, use URL names and `resolve_url` ; )